packages/http-client/src/__tests__/remote-index-api.test.ts

```diff
@@ -29,7 +29,7 @@ test('model in query', async () => {
   ;(indexApi as any)._fetchJson = fauxFetch
   const result = await indexApi.query({ model: MODEL, first: 5 })
   expect(result).toEqual(EMPTY_RESPONSE)
-  expect(fauxFetch).toBeCalledWith(new URL(`https://example.com/collection?model=${MODEL}&first=5`))
+  expect(fauxFetch).toBeCalledWith(new URL(`https://example.com/collection/?model=${MODEL}&first=5`))
```

Comment:
this change seems surprising to me?  I'd expect the query params string to come directly on the `collection` endpoint, not after a slash to a new sub-endpoint?